.github/workflows/static_code_analysis.yml

```diff
@@ -23,6 +23,13 @@ jobs:
       - run: echo "Branch ${{ github.ref }} of repository ${{ github.repository }}."
       - name: Check out repository code
         uses: actions/checkout@v4
+      - name: Install helm
+        uses: azure/setup-helm@v4.2.0
+        with:
+          version: latest
+        id: install
+      - name: Run helm and template locally
+        run: helm template charts/kubearchive/ --output-dir charts/generated
```

Comment:
would it make more sense to run `helm lint` instead of `helm template`?

https://helm.sh/docs/helm/helm_lint/

Reply:
I don't think it does in the context of yamllint, which is what we are trying to run.

We're not linting the chart itself (which is what helm lint does), we're linting the output of the helm templating.

Reply:
if the decision is made to switch to `helm lint` the helm stuff should get moved to its own job

Reply:
Unless `helm lint` runs `yamllint` under the covers, I think we'll always need to do this.

We have to run `helm` here to get the fully-templated file for `yamllint`.  I don't see that ever changing, unless we abandon `helm` completely.  This exercise already found one problem, which shows that it's important.